project/SparkBuild.scala

```diff
@@ -415,8 +415,7 @@ object SparkBuild extends PomBuild {
   /* Protobuf settings */
   enable(SparkProtobuf.settings)(protobuf)
 
-  // SPARK-14738 - Remove docker tests from main Spark build
-  // enable(DockerIntegrationTests.settings)(dockerIntegrationTests)
+  enable(DockerIntegrationTests.settings)(dockerIntegrationTests)
```

Comment:
Please proceed this as an independent PR, @yaooqinn .

Reply:
I wonder why it was commented out before. The docker-integration-tests is an optional profile disabled by default. If this is commented out, when will `guava-18.0` be activated? Only thing I can guess is that the guava code is untouched by spotify at all.

As for now, if we separate this to a post-proceed PR, we are not able to pass this one?

```java
[info] org.apache.spark.sql.jdbc.MariaDBKrbIntegrationSuite *** ABORTED *** (716 milliseconds)
[info]   java.lang.NoClassDefFoundError: com/google/common/collect/MultimapBuilder
[info]   at com.github.dockerjava.core.DefaultDockerCmdExecFactory$DefaultWebTarget.<init>(DefaultDockerCmdExecFactory.java:57)
```

Reply:
```
enable(KubernetesIntegrationTests.settings)(kubernetesIntegrationTests)
```

It looks like KubernetesIntegrationTests is also activated by default.